src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +270,70 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool All<TNode>(SyntaxList<TNode> items, Func<TNode, bool> predicate)
+            where TNode : SyntaxNode
+        {
+            bool result = true;
+            for (int i = 0; i < items.Count; i++)
+            {
+                result &= predicate(items[i]);
+            }
+
+            return result;
+        }
+
+        private static bool IsAutoAccessorList(SyntaxToken token)
+            => token.Parent is AccessorListSyntax accessorList &&
+                All(accessorList.Accessors, a => a.Body == null);
+
+        private static bool IsAutoAccessorList(SyntaxNode? node)
+            => node?.Parent is AccessorListSyntax accessorList &&
+                All(accessorList.Accessors, a => a.Body == null);
+
+        private static bool HasInitializer(BasePropertyDeclarationSyntax basePropertyDeclaration)
+            => basePropertyDeclaration is PropertyDeclarationSyntax property && property.Initializer != null;
+
+        private static bool IsAccessorListFollowedByInitializer(SyntaxToken token)
```

Comment:
>IsAccessorListFollowedByInitializer [](start = 28, length = 35)

Token cannot be an accessor list

Reply:
I renamed it to "IsParentAccessorListFollowedByInitializer". Please confirm if it is ok for you before committing.
I preferred to stay with `BasePropertyDeclarationSyntax` for potential future changes in the language, even if currently indexers can't be auto-implemented.

Reply:
> I renamed it to "IsParentAccessorListFollowedByInitializer". Please confirm if it is ok for you before committing.

I suggest leaving the work to get the parent to the consumers, the function checks a SyntaxNode and the name stays the same.

---
In reply to: [531009152](https://github.com/dotnet/roslyn/pull/49495#discussion_r531009152) [](ancestors = 531009152)

Reply:
I just followed the same "style" of the other existing methods `LineBreaksAfterOpenBrace` and `LineBreaksAfterCloseBrace` which used to get the token and work on the parent.
Anyway, I am changing that.